src/components/graph/graph.css

```diff
@@ -0,0 +1,15 @@
+.Graph {
+  display: flex;
+  justify-content: right;
+  margin-right: 2em;
+  max-width: 300px;
+  max-height: 300px;
+}
+
+.Container {
```

Comment:
Some of our class names are quite generic. This can cause unexpected behaviour when adding the `Container` class elsewhere. I believe we're currently using the `emotion` library to avoid this. Let's make sure the CSS compiles correctly (this can be as simple as checking the HTML when running the app).

Reply:
changed the naming to GraphContainer